addons/mail/sfu_server/README.md

```diff
@@ -0,0 +1,158 @@
+# Odoo: Discuss SFU
+
+## Overview
+
+Contains the code for an SFU (Selective Forwarding Unit) server 
+to be used alongside the Odoo Discuss module. The SFU server is responsible for
+handling the WebRTC connections between users and providing channels to coordinate
+these connections.
+
+## Prerequisites
+
+- An Odoo server with the Discuss module installed
+- [Node.js](https://nodejs.org/)
+
+## Before deployment
+
+Build the client bundle
+
+ ```bash
+     npm install
+     npm run build
+ ```
+
+Once the bundle is built, it should be added to the libraries of the Odoo server that will use
+this version of the server.
+
+## Deployment
+
+1. Install dependencies.
+    ```bash
+        npm ci -omit=dev
+    ```
+2. Run the server.
+    ```bash
+        npm PROXY=SSL PUBLIC_IP=134.123.222.111 AUTH_KEY=u6bsUQEWrHdKIuYplirRnbBmLbrKV5PxKG7DtA71mng= run start
+    ```
+
+The available environment variables are:
+
+- **PUBLIC_IP** (required): used to establish webRTC connections to the server
+- **AUTH_KEY** (required): the base64 encoded encryption key used for authentication
+- **HOSTNAME**:  HTTP/WS hostname, defaults to "0.0.0.0" (listen on all interfaces)
```

Comment:
I think HOSTNAME and listen interface are different parameters. Otherwise we get:
```
[HTTP] - http listening on sfu01-nim:80
```
That doesn't really make sense. You want to listen to 127.0.0.1, and set the hostname to sfu01-nim.odoo.com

Reply:
in the node http code, this param is called `hostname`, in the doc it is this host param: https://nodejs.org/api/net.html#serverlistenport-host-backlog-callback which points to https://nodejs.org/api/net.html#netcreateconnectionport-host-connectlistener

I think that in your case it is `localhost`/`127.0.0.1` not `sfu01-nim.odoo.com`